.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!--
+     Before submitting a Pull Request, please ensure you've done the following:
+     - 📖 Read our Contributing Guide & Coding Standards: https://github.com/4chain-ag/code-standards-bsv/blob/main/.github/CODE_STANDARDS.md
+     - 📖 Read the short Code of Conduct: https://github.com/4chain-ag/code-standards-bsv/blob/main/.github/CODE_OF_CONDUCT.md
+     - 👷‍♀️ Created small PRs. In most cases this will be possible.
+     - ✅ Provided tests for your changes.
+     - 📝 Used conventional commits.
+     - 📗 Update any related documentation.
+-->
+
+# Pull Request Template
+
+## What type of PR is this?
+
+- [ ] Refactor
+- [ ] Feature
+- [ ] Bug Fix
+- [ ] Security Fix/Improvement
+- [ ] Optimization
+- [ ] Documentation Update
```

Comment:
We want to require to use conventional commits rules in PR title also, so type of change is actually part of the PR title.

Reply:
But, maybe it's worth to mention that title of the PR should also match conventional commits pattern?